waspls/src/Wasp/LSP/Handlers.hs

```diff
@@ -0,0 +1,118 @@
+module Wasp.LSP.Handlers
```

Comment:
Let's also reorganize this file so it is read like story + it has nice names + it has some basic comments that help quickly understand more complex stuff without going through the LSP documentation.